tests/test_management_commands.py

```diff
@@ -149,12 +321,12 @@ def test_basic_invocation_and_force_param(self):
         expected_msg = "After es_create --force, the movies index should have two versions"
         self.assertEqual(len(available_version_ids), 2, expected_msg)
 
-    def test_all_and_force_params(self):
-        movies_index_model, _, __ = self._check_basic_setup_and_get_models("movies")
+    def test_all_and_force_flags(self):
+        movies_index_model, _, __ = self.check_basic_setup_and_get_models("movies")
```

Comment:
I don't think I've ever seen the underscore, double_underscore.. I'm assuming it's just another unnamed/unused variable right?

Reply:
Good eyes - this is a mistakenly applied habit from OCaml, when unpacking tuples that have more than one variable that are not needed still need to be named differently. I'll update.

Reply:
updated in edfad8ff